server/src/main/java/io/kroki/server/Server.java

```diff
@@ -72,7 +79,59 @@ static void start(Vertx vertx, VertxOptions vertxOptions, JsonObject config, Han
     HttpServer server = vertx.createHttpServer(serverOptions);
     Router router = Router.router(vertx);
     BodyHandler bodyHandler = BodyHandler.create(false).setBodyLimit(config.getLong("KROKI_BODY_LIMIT", BodyHandler.DEFAULT_BODY_LIMIT));
-
+    HttpClient delegatorHttpClient = vertx.httpClientBuilder().withRedirectHandler(resp -> {
```

Comment:
could you please add a link to the original code and explain why we are not using the default redirectHandler (i.e., why we are not using `setFollowRedirects`):

https://github.com/eclipse-vertx/vert.x/blob/78ca74e30d8e0b7b8950c4c4030b45224c2a7e0a/vertx-core/src/main/java/io/vertx/core/http/impl/DefaultRedirectHandler.java#L25

Reply:
Added comment